content/opennox/mod/_index.en.md

```diff
@@ -5,4 +5,6 @@ weight: 6
 
 OpenNox provides a lot of tooling for modding the game. Some of it is still in development.
 
-{{% children sort="weight" %}}
\ No newline at end of file
+The `noxtools` binary is located in the [opennox/libs](https://github.com/opennox/libs) repository under the `noxtools` folder. To build it, run `go build` from the `noxtools` folder.
```

Comment:
One may also run this to install the binary to `~/go/bin`:
```
go install github.com/opennox/libs/cmd/noxtools@latest
```
or to get the binary in the current folder:
```
go build ./cmd/noxtools
```